homeassistant/package_constraints.txt

```diff
@@ -2,6 +2,7 @@
 
 aiodhcpwatcher==1.0.0
 aiodiscover==2.0.0
+aiodns==3.1.1
```

Comment:
This is already going to be installed anyways since its a dep from `dhcp` via `default_config`